playwright.config.cjs

```diff
@@ -26,6 +26,7 @@ module.exports = defineConfig({
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
   reporter: 'html',
   /* Shared settings for all the projects below. See https://playwright.dev/docs/api/class-testoptions. */
+  timeout: 60000,
```

Comment:
Not sure if this increases/decreases from the previous setting. If the latter, we need to check if existing tests fail.

Reply:
This is an increase from previous setting as I have one test that takes a longer path. Default setting is 30 seconds I doubled it to be on the safe side. documentations here: https://playwright.dev/docs/test-timeouts